routes/cube_routes.js

```diff
@@ -2633,6 +2634,19 @@ router.get(
   }),
 );
 
+router.get(
+  '/api/cubecardtags/:id',
+  util.wrapAsyncApi(async (req, res) => {
+    const cube = await Cube.findOne(buildIdQuery(req.params.id)).lean();
+    const tags = cubeCardTags(cube);
```

Comment:
@dekkerglen this seems a bit painful to do each request, would this make sense to do once and stored when a cube is saved/updated?
Maybe safest/simplest to do the slow way though.

Reply:
I'll take a look of this route is called too often, but it probably shouldn't be right now and it probably is not too expensive I would think?
Caching the value would add a lot of complexity, where I would guess it is not worth for the potential benefit it might bring?